fsys_test.go

```diff
@@ -0,0 +1,135 @@
+package skeletonkit_test
+
+import (
+	"embed"
+	"flag"
+	"io"
+	"path/filepath"
+	"strings"
+	"testing"
+	"text/template"
+
+	"github.com/gostaticanalysis/skeletonkit"
+	"github.com/tenntenn/golden"
+)
+
+type appInfo struct {
+	Name       string
+	ModulePath string
+}
+
+type errMap struct {
```

Comment:
I propose use `errs` instead of `errMap` because we can write short code in a test case. By defining the type in test function, it does not conflict other definition.